### DISCUSSION TITLE
Graphical Arch Install

### DISCUSSION THREAD

**Discussion by Phil995511** [archlinux/archinstall#1786] [Ideas]
Hello,

Would it be possible for you to develop a graphical Arch installation tool ?

Best regards.

**Comment by dylanmtaylor** [comment]
There was one at one point. https://github.com/Torxed/archinstall_gui

**Reply by Torxed** [reply]
Would have loved to make that an actual reality, but just maintaining `archinstall` alone is enough for the foreseeable future.

But I liked the concept of making it web-browser based, opening up the possibility for remote deployments and other shenanigans.

**Reply by ghost** [reply]
> Would have loved to make that an actual reality, but just maintaining `archinstall` alone is enough for the foreseeable future.
> 
> But I liked the concept of making it web-browser based, opening up the possibility for remote deployments and other shenanigans.

I think this is a perfect approach 

It will be nice something like [agama](https://github.com/openSUSE/agama) the new installer from openSUSE

**Comment by Phil995511** [comment]
I discovered a few days ago RebornOS, which is based on Arch Linux. I really like it because it retains the spirit of Arch Linux, unlike for example to Manjaro and its home made kernels. It is this type of installer (and simpler administration via a graphical interface) that Arch should offer us... I hope that Arch will have some good surprises in store for us and evolve in the right direction in the near future, because this OS really has a very strong potential and I would migrate to it if it was a little more userfreindly :)

Thank you for your commitment and for your work.

Best regards.
